core/controllers/creator_dashboard_test.py

```diff
@@ -79,14 +109,14 @@ def test_logged_out_homepage(self):
     def test_notifications_dashboard_redirects_for_logged_out_users(self):
         """Test the logged-out view of the notifications dashboard."""
         response = self.get_html_response(
-            '/notifications_dashboard', expected_status_int=302)
+            '/notifications', expected_status_int=302)
```

Comment:
should this be 301, and ditto elsewhere?

Reply:
@seanlip Ah, that's for when the user is logged out. They should be redirected to the home-page in that case. This redirection is done by [this method](https://github.com/oppia/oppia/blob/b3d321a6f17439ff9180095bd9458b06002b7196/core/controllers/acl_decorators.py#L1523). Changing this method's redirect to a permanent redirect would affect every other logged-out redirection. I'll make that change if that's what needs to be done.

/cc @kevintab95

Reply:
Oh I see. Yep please keep it as 302 -- thanks!